packages/node_modules/@ciscospark/react-component-button-controls/src/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import Button from '@ciscospark/react-component-button';
+
+import styles from './styles.css';
+
+export default function ButtonControls(props) {
+  const {
+    buttons,
+    showLabels
+  } = props;
+
+  const buttonsRendered = buttons.map((button) =>
+    <div className={classNames(`button-control-item`, styles.controlItem)} key={button.label}>
+      <Button
+        {...button}
+        buttonClassName={classNames(button.buttonClassName, styles.button)}
+        label={showLabels && button.label}
+        labelPosition={showLabels && `bottom`}
+      />
+    </div>
+  );
+
+  return (
+    <div className={classNames(`controls-container`, styles.controlContainer)}>
+      {buttonsRendered}
+    </div>
+  );
+}
+
+ButtonControls.propTypes = {
+  buttons: PropTypes.arrayOf(PropTypes.shape({
+    label: PropTypes.string,
```

Comment:
Like a `buildButton({label, ...button})` method? I'm not sure it's necessary since these are the base props for a normal button anyways?